packages/elastic-safer-lodash-set/scripts/update.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env sh
+
+# Elasticsearch B.V licenses this file to you under the MIT License.
+# See `packages/elastic-safer-lodash-set/LICENSE` more information.
+
+set -e
+
+source ./scripts/_get_lodash.sh
+
+all_files=$(cd lodash && ls)
+modified_lodash_files=(_baseSet.js)
+
+# Get fresh copies of all the files that was originally copied from loadsh,
+# expect the ones in the whitelist
+for file in $all_files
+do
+  if [[ ! "${modified_lodash_files[@]}" =~ "${file}" ]]
+  then
+    cat scripts/license-header.txt > "lodash/$file"
+    echo "/* eslint-disable */\n" >> "lodash/$file"
```

Comment:
__note:__ this actually leaves a `\n` literal in the generated files, I'm wondering if you need to use [`printf` or something](https://github.com/koalaman/shellcheck/wiki/SC2028#rationale).

Reply:
Weird, the output looks fine when I view it... maybe it's something weird with my shell that accidentally makes it work. I'll take a look

Reply:
Ah, after changing the hash-bang from `sh` to `bash`, it now prints the literal `\n` as you said. I'll fix it 👍

Reply:
Fixed 👍